src/test/java/TestEditRulePopup.java

```diff
@@ -26,44 +29,65 @@ void setUp() throws Exception {
 		addpopup = new EditRulePopup(new CodeQualityRule("", "LAA < 5", false, false));
 	}
 
+	/**
+	 * Tests getConditon() method
+	 */
 	@Test
 	void testGetCondition() {
 		addpopup.getCondition();
 	}
-
+	
+	/**
+	 * Tests getValue() method
+	 */
 	@Test
 	void testGetValue() {
 		addpopup.getValue();
 	}
-
+	
+	/**
+	 * Tests getComparison method

```

Comment:
please update all the docs to explain what the method being tested is, as I did before